clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -47,10 +47,6 @@
                 "command": "cobol-lsp.cpy-manager.fetch-copybook",
                 "title": "Fetch copybook"
             },
-            {
```

Comment:
Do we need this? I mean we can always get the user profile from settings rather than creating a service

Reply:
I meant , "cobol-lsp.cpy-manager.profiles": "profileName"